website/events-listener/src/index.ts

```diff
@@ -1,17 +1,17 @@
 import { GearApi } from '@gear-js/api';
+import { logger } from '@gear-js/common';
 
 import { listen } from './events';
 import { KafkaProducer } from './producer';
-import { logger } from './logger';
 import config from './config';
 import { restartIfNeeded, setRestartNeeded } from './lifecycle';
 import { changeStatus } from './healthcheck';
 
-const log = logger('Main');
+export const EVENTS_LISTENER = 'EVENTS_LISTENER';
 
 const main = async () => {
   while (true) {
-    log.info('Starting...');
+    logger.info(`${EVENTS_LISTENER} Starting...`);
```

Comment:
May be it could just be `Main`?

Reply:
@osipov-mit there is also the same thing. Try to imagine that you want to know every service in running `terminal`.

But in `terminal` you have only `Main` name it doesn't tell us what the service is running exactly.